admin/bin/qserv-admin.py

```diff
@@ -306,6 +313,27 @@ def _parseCreateNode(self, tokens):
                                 int(options.get('port', 0)), options['state'])
         self._css.addNode(options['nodeName'], params)
 
+    def _parseCreateChunk(self, tokens):
+        """
+        Subparser - handles all CREATE CHUNK requests.
+        Throws css.CssError, _ToolError.
+        """
+        if len(tokens) != 3:
+            raise _IllegalCommandError("Unexpected number of arguments.")
+
+        (dbTbName, chunkIdStr, nodeName) = tokens
+        if '.' not in dbTbName:
+            raise _IllegalCommandError("Invalid argument '{0}', should be <dbName>.<tbName>".format(dbTbName))
+
+        (dbName, tbName) = dbTbName.split('.')
```

Comment:
This was a "cut-and-paste" pattern found in 4 other places within the same file. I chose not to refactor the application, and just leave it all as is. Ideally, there database/table name translation should be handled in a dedicated function/method in which all possible syntax errors would be taken care of. Thank you for bringing this up!